### ISSUE TITLE
A param of type 'List' is breaking camelCase()

### ISSUE THREAD

**Issue by chrisbuttery** [icelab/formalist-compose#10]
It seems a `List` is being passed through the `camelCase` instead of a `string`.

Closed PR: https://github.com/icelab/formalist-compose/pull/9#issuecomment-205170272

![screen shot 2016-04-04 at 5 16 40 pm](https://cloud.githubusercontent.com/assets/121675/14240594/74ace8bc-fa89-11e5-968d-29cf8025e1e3.png)

**Comment by makenosound** [comment]
@chrisbuttery Yeah, so the likely cause is that the formalist AST doesn’t match what we’re expecting somewhere. Can you work out which field/section/group etc combination is causing that error?

I think `throw`ing an error when the value isn’t a `String` here is totally reasonable, makes it clearer what’s going on.

**Comment by chrisbuttery** [comment]
@makenosound I've just addressed this in PR https://github.com/icelab/formalist-compose/pull/13

**Comment by makenosound** [comment]
@chrisbuttery Cool, thanks!

**Comment by timriley** [comment]
We should keep this open to make sure we work out and prevent the original issue about the wrong data being passed to the camel case function, yes?

**Comment by makenosound** [comment]
@timriley I think it was just old AST data.

**Comment by timriley** [comment]
Ah, OK then!

**Comment by chrisbuttery** [comment]
@makenosound @timriley Yeah I believe it was. I'll leave open for now.
